From: =?utf-8?b?0KDRg9GB0LvQsNC9INCY0LbQsdGD0LvQsNGC0L7Qsg==?=
 <lrn1986@gmail.com>
Date: Thu, 20 Sep 2018 05:20:24 +0000
Subject: tests W32: include process.h for getpid()

At least, MinGW has POSIX names. No idea if this works in MSVC.

Origin: upstream, 2.59.0, commit:d2c9543f2e674d20e72144122b7409c31f08c7e0
---
 tests/mapping-test.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/tests/mapping-test.c b/tests/mapping-test.c
index 3eb90fa..27c6d83 100644
--- a/tests/mapping-test.c
+++ b/tests/mapping-test.c
@@ -25,6 +25,9 @@
 #ifdef G_OS_UNIX
 #include <unistd.h>
 #endif
+#ifdef G_OS_WIN32
+#include <process.h>
+#endif
 
 static gchar *dir, *filename, *displayname, *childname;
 
